h2o-genmodel/src/main/java/hex/genmodel/algos/gam/GamMojoModelBase.java

```diff
@@ -60,8 +60,7 @@
 
   @Override
   public double[] score0(double[] row, double[] preds) {
-    if (_meanImputation)
-      imputeMissingWithMeans(row);  // perform imputation for each row
+    imputeMissingWithMeans(row);  // perform imputation for each row
```

Comment:
should also remove _meanImputation since it is unused now

Reply:
Sure.